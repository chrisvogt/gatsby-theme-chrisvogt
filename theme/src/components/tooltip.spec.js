import React from 'react'
import renderer from 'react-test-renderer'

import Tooltip from './tooltip'

const DemoContent = () => (
  <div className='demo-content'>
    <span>I'm inside a tooltip!</span>
  </div>
)

describe('Tooltip', () => {
  it('matches the snapshot', () => {
    const tree = renderer
      .create(
        <Tooltip
          tooltip={DemoContent}
          hideArrow
        >
          Trigger element
        </Tooltip>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
