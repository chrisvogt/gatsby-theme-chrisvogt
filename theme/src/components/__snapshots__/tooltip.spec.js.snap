// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip matches the snapshot 1`] = `
<span>
  Trigger element
</span>
`;
